docs/database/core-api-query-indexes.sql

```
@@ -0,0 +1,23 @@
+FROM user_ind_columns
+WHERE table_name IN ('PERFORMANCE_SEATS', 'ORDER_SEATS')
+ORDER BY table_name, index_name, column_position;
+
+-- 3. 동일 목적의 인덱스가 없을 때만 실행한다.
+-- 좌석 선택 검증의 performance_id + seat_id 조건을 인덱스 탐색으로 처리하고 중복 행을 방지한다.
+CREATE UNIQUE INDEX uk_performance_seats_performance_seat
+    ON performance_seats (performance_id, seat_id);
+
+-- 주문 상세의 order_seats 조인을 order_id 기준 인덱스 탐색으로 처리한다.
+CREATE INDEX idx_order_seats_order_id
+    ON order_seats (order_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)